src/sampler/index.js

```diff
@@ -19,8 +19,10 @@ export default function Sampler({ data, mappings, exportCallback }) {
     const searchQuery = useSearchQuery();
     const highlighted = useHighlight();
 
+    const cachedView = localStorage.getItem('view');
+
     const [flameData, setFlameData] = useState(null);
-    const [view, setView] = useState(VIEW_ALL);
+    const [view, setView] = useState((cachedView === 'VIEW_FLAT' ? VIEW_FLAT : cachedView === 'VIEW_SOURCES' ? VIEW_SOURCES : VIEW_ALL));
```

Comment:
There is no way to retrieve the actual constant object type from `VIEWS` array and from `/views` file because the `localStorage#getItem` returns a string. In order to make this happen, it needs to specify variables for each element of `VIEWS` array, but I will not doing this as it will breaks some things.